text/0000-trait-alias.md

```diff
@@ -0,0 +1,266 @@
+- Feature Name: Trait alias
+- Start Date: 2016-08-31
+- RFC PR:
+- Rust Issue:
+
+# Summary
+[summary]: #summary
+
+Traits can be aliased the same way types can be aliased with the `type` keyword.
```

Comment:
Is this summary still accurate? I think it should be s/`type` keyword/`trait` keyword/, no?